docs/source/use_with_pytorch.mdx

```diff
@@ -201,22 +201,22 @@ You must use a `BatchSampler` if you want the transform to be given full batches
 ### Stream data
 
 Loading a dataset in streaming mode is useful to progressively download the data you need while iterating over the dataset.
```

Comment:
```suggestion
Loading a dataset in streaming mode allows one to iterate over the dataset without downloading it on disk.
```
incorrect streaming description from past again 👻

There is the same in line 152 (in Optimize data loading section) but I can't fix it there.